jira_agile_metrics/config_test.py

```diff
@@ -256,23 +252,22 @@ def test_config_to_options_maximal():
         'queries': [{'jql': '(filter=123)', 'value': 'Team 1'},
                     {'jql': '(filter=124)', 'value': 'Team 2'}],
         'query_attribute': 'Team',
-                
+
         'backlog_column': 'Backlog',
         'committed_column': 'Committed',
-        'final_column': 'Test',
```

Comment:
Again - changes a test that would have passed previously.

Reply:
I would be okay to leave it there. However, it's not needed and would be ignored. I find it confusing to have «dead» config in tests. Maybe it makes sense to add a comment noting that this is a legacy configuration parameter.